browser/src/app/TilesMiddleware.ts

```diff
@@ -1579,7 +1583,8 @@ class TileManager {
 			if (
 				coords.part === part &&
 				coords.mode === mode &&
-				invalidatedRectangle.intersectsRectangle(tileRectangle)
+				(invalidatedRectangle.intersectsRectangle(tileRectangle) ||
+					(calc && coords.scale !== scale)) // In calc, we invalidate all tiles with different zoom levels.
```

Comment:
I'm confused as to why this problem would be specific to calc?

Reply:
Because we adjust the twips position for Calc. We cannot use the twips (from core side) directly. But for other files, we can calculate the twips' position for different scales. Please see "getTileTwipsSheetAreaFromPrint" function and related things for more info.

Reply:
We can also do the calculation for Calc but it would be expensive for this case.

Reply:
I'll take your word for it :) I might be nice to have a comment here briefly explaining why this is done for calc alone.